server/controllers/userInputController.ts

```diff
@@ -68,27 +72,10 @@ const createUserInputController = (
       logger.debug('New Linkta flow created:', newLinktaFlow);
 
       if (newLinktaFlow) {
-        const { _id, userInputId, nodes, edges } = newLinktaFlow;
-
-        const mappedNodes = nodes.map((node: CustomNode) => ({
-          ...node._doc,
-          id: node._id,
-          _id: undefined,
-        }));
-
-        const mappedEdges = edges.map((edge: CustomEdge) => ({
-          ...edge._doc,
-          id: edge._id,
-          _id: undefined,
-        }));
-
-        res.locals.linktaFlow = {
-          id: _id,
-          userInputId,
-          nodes: mappedNodes,
-          edges: mappedEdges,
-        };
+        res.locals.linktaFlowId = newLinktaFlow._id;
```

Comment:
Reason for this change pls see PR #364

Reply:
I just looked through #364 and approved that, this all looks great, thank you.

Reply:
Thank you Ted!